.gitignore

```
@@ -22,3 +22,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+tmp/*
```

This should be extended to ignore `modules/backend/tmp/main`

```suggestion
**/tmp/*
```